src/schema/vosummary.xsd

```diff
@@ -187,6 +187,8 @@
                               <xsd:sequence>
                                 <xsd:element name="URL" type="xsd:string"/>
                                 <xsd:element name="DefaultUnixUser" type="xsd:string"/>
+                                <xsd:element name="Description" type="xsd:string"/>
+                                <xsd:element name="Subject" type="xsd:string"/>
```

Comment:
```suggestion
                                <xsd:element minOccurs="0" name="Description" type="xsd:string"/>
                                <xsd:element minOccurs="0" name="Subject" type="xsd:string"/>
```

Reply:
This one's not necessary -- if it's not specified in the yaml, the xml fields still show up, they're just empty.

Reply:
Oh gotcha